config/routes.rb

```diff
@@ -3,4 +3,6 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get 'auth/google_oauth2/callback', to: 'sessions#create'
+  get '/dashboard', to: 'dashboard#show'
```

Comment:
No need to address in this PR, but let's revisit front end route names as a group to decide exactly what we want them to be from the user's perspective.

Reply:
@exasperlnc @SMcPhee19

Reply:
Sounds like a plan! We can do it during tomorrow's Standup? I'll add it to the miro board!